### PR TITLE
expat: minor upgrade to 2.4.9 to fix CVE-2022-40674

### PR DESCRIPTION
**Pull request by aditjha-msft**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Fix CVE-2022-40674

###### Change Log  

- Change
- Change
- Change

###### Does this affect the toolchain?  

**YES/NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2022-40674

###### Test Methodology

- BuddyBuild was able to build expat, tested on Mariner VM and rpm installs worked.
- Build Toolchain passed

### CODE REVIEW

File: SPECS/expat/CVE-2022-40674.patch [hunk 1]
```diff
@@ -0,0 +1,49 @@
+From 4a32da87e931ba54393d465bb77c40b5c33d343b Mon Sep 17 00:00:00 2001
```
Comment (christopherco) [inline SPECS/expat/CVE-2022-40674.patch:1]: Is the CVE patch still needed if we upgrade expat to 2.4.9?
From the release notes, it seems like the patch is already incorporated into 2.4.9.

**Review by christopherco — CHANGES_REQUESTED** [review]
Looks like expat is part of the raw toolchain build (toolkit/scripts/toolchain/container/toolchain_build_in_chroot.sh)

We should update this as well

File: SPECS/expat/CVE-2022-40674.patch (hunk 1 above)
Comment (aditjha-msft) [reply SPECS/expat/CVE-2022-40674.patch]: deleted patch file

File: toolkit/resources/manifests/package/toolchain_x86_64.txt [hunk 2]
```diff
@@ -119,7 +119,7 @@ glibc-nscd-2.28-24.cm1.x86_64.rpm
 glibc-tools-2.28-24.cm1.x86_64.rpm
 glib-debuginfo-2.58.0-9.cm1.x86_64.rpm
 glib-devel-2.58.0-9.cm1.x86_64.rpm
-glib-schemas-2.58.0-9.cm1.x86_64.rpm
+glib-sx86_64chemas-2.58.0-9.cm1.x86_64.rpm
```
Comment (neha170) [inline toolkit/resources/manifests/package/toolchain_x86_64.txt:122]: Looks like a typo here

**Review by nicogbg — APPROVED** [review]

**Review by neha170 — APPROVED** [review]

**Review by neha170 — COMMENTED** [review]
Please update the pipeline build id in the description with the id for the full build (including toolchain), thanks

**Pull request merged**